ros/src/computing/perception/detection/lib/image/kf/src/kf.cpp

```diff
@@ -611,7 +615,7 @@ void doTracking(std::vector<cv::LatentSvmDetector::ObjectDetection>& detections,
 	{
 		if (add_as_new_indices[i])
 		{
-			initTracking(objects[i], kstates, detections[i], image, colors);
+			initTracking(objects[i], kstates, detections[i], image, colors, _ranges[i]);
```

Comment:
@amc-nu ,
(Sorry for contacting when you are busy...)

Could you please check whether the index added here is correct?
I found that /image_obj_tracked didn't contain range information (it was always zero) and added assign statement, but I'm not sure this "i"th range value is correspond to one for detection[i].

I would appreciate if you check this modification and merge this PR if it's no problem.